packages/forklift-console-plugin/src/modules/Plans/views/details/utils/hasPlanIncludedCompleteRunningVMs.ts

```diff
@@ -1,6 +1,6 @@
 import { V1beta1Plan } from '@kubev2v/types';
 
-export const hasPlanEditable = (plan: V1beta1Plan) => {
+export const hasPlanIncludedCompleteRunningVMs = (plan: V1beta1Plan) => {
```

Comment:
nitpick - naming suggestions: `hasASuccessfullyMigratedVM`, `hasACompleteRunningVM`, or `hasSomeCompleteRunningVMs`

The function is operating on the plan argument, so I don't think `plan` needs to be part of the function name.